src/input/mpegts/tvhdhomerun/tvhdhomerun_frontend.c

```diff
@@ -325,6 +330,9 @@ static void tvhdhomerun_device_open_pid(tvhdhomerun_frontend_t *hfe, int pid) {
 
   tvhdebug("tvhdhomerun", "current pfilter: %s", pfilter);
 
+  /* make sure the pid maps to a max of 0x1FFF, API will reject the call otherwise */
+  pid = (pid & 0x1FFF);
```

Comment:
Good idea to log an error in that case, even if it is an edge case which we will probably never see. There is going to be a migration on my DVB-C network and I need mux detection to work, hence this fix.